db/migrate/20170201155426_add_highstate_to_minions.rb

```diff
@@ -0,0 +1,5 @@
+class AddHighstateToMinions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :minions, :highstate, :boolean
```

Comment:
Maybe add an `after: :role`, so we keep `created_at` and `updated_at` the last ones.